components/org.wso2.carbon.identity.governance/src/main/java/org/wso2/carbon/identity/governance/internal/service/impl/otp/OTPGeneratorImpl.java

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright (c) 2022, WSO2 LLC. (http://www.wso2.com).
```

Comment:
Shall we update the license headers?

Reply:
This commit is from 2022, should we update the year in this file ?

Reply:
Changed to 2023 in 37a466ec8bc36eb2f5a3a2d160ff5a557ed99a9d